### DISCUSSION TITLE
Notify of container image updates

### DISCUSSION THREAD

**Discussion by deanfourie1** [louislam/dockge#655] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

The ability to indicate or notify if a container image update is available.

### ✔️ Solution

A visual indicator that either state "Update available" or "Up to Date"

### ❓ Alternatives

Checking each container for updates individually. :(

**Comment by tismofied** [comment]
#239
